src/zenml/zen_server/routers/projects_endpoints.py

```diff
@@ -511,6 +511,48 @@ def list_project_pipelines(
     )
 
 
+@router.get(
+    "/{project_name_or_id}" + PIPELINES,
+    response_model=List[PipelineRunResponseModel],
+    responses={401: error_response, 404: error_response, 422: error_response},
+)
+@handle_exceptions
+def list_runs(
+    project_name_or_id: Union[str, UUID],
+    stack_id: Optional[UUID] = None,
+    run_name: Optional[str] = None,
```

Comment:
I think we had a problem on develop this week where we realized that our list endpoint need to have an argument called `name` (instead of the `run_name` here. I'm guessing that's also relevant here @bcdurak?

Reply:
Would make sense because else our serialization at the rest_zen_store will try filtering by name but the endpoint has run_name as the keyword arg. Will include this as well

Reply:
done